popup.el

```diff
@@ -934,7 +934,8 @@ Pages up through POPUP."
                          (cursor-color popup-isearch-cursor-color)
                          (keymap popup-isearch-keymap)
                          callback
-                         help-delay)
+                         help-delay
+			 isearch-electric)
```

Comment:
Can you fix the indentation here? Probably should use spaces instead of tabs. Thanks!